src/transformStateWithClones.js

```diff
@@ -7,7 +7,25 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateHistory = [];
+  let currentState = { ...state };
```

Comment:
The variable `currentState` is a good example of proper naming, as it clearly describes that it is a copy of the state that will be transformed.